helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -0,0 +1,52 @@
+additional: {}
+# |-
+#   kind: NetworkPolicy
+#   apiVersion: networking.k8s.io/v1
+#   metadata:
+#     name: example-np
+#     namespace: default
+#   spec:
+#     policyTypes:
+#       - Ingress
+#       - Egress
+#     podSelector:
+#       matchLabels:
+#         foo: bar
+#     ingress: {}
+#     egress: {}
+harbor:
+  enable: true
+  chartmuseum:
+    httpsIPs:
+      - "0.0.0.0/0" # S3
+  core:
+    httpsIPs:
+      - "0.0.0.0/0" # S3, openid
+  database:
+    type: internal
+    internal: []
+      # - ingressNSMatchLabels: {}
+      #   ingressNSPodMatchLabels: {}
+    external: []
+      # - egressNSMatchLabels:
+      #     kubernetes.io/metadata.name: <postgres namespace>
+      #   egressNSPodMatchLabels: {}
+      #   ports:
+      #    - 5432
```

Comment:
Is this correct?
```suggestion
    internal: []
      # - ingressNSMatchLabels: {}
      #   ingressPodMatchLabels: {}
    external: []
      # - egressNSMatchLabels:
      #     kubernetes.io/metadata.name: <postgres namespace>
      #   egressPodMatchLabels: {}
      #   ports:
      #    - 5432
```

Reply:
I wanted to make a bit more clear that that pod selector is part of the ns mach label.

Reply:
Ok. Since they are part of the same list item I think it's probably fine without the `NS` for the pod labels. But if you think it makes it more clear then I'm ok to keep it.

Reply:
Yeah i think i remove it as you say they are part of the same list item.